manifests/init.pp

```diff
@@ -1,5 +1,20 @@
 # Class: certmonger
-class certmonger {
+class certmonger(
+  $manage_dbus_service = false
+) {
+  # DBus package update from 7.4 to 7.5 breaks
+  # The certmonger service, so dbus must get a
+  # restart prior to start certmonger.
+  # The official solution for dbus is either
+  # restart the service, or reboot the machine
+  if $manage_dbus_service {
+    service { 'dbus':
+      ensure => running,
+      enable => true,
+      restart => '',
```

Comment:
Not really usefuly to set this, according to the doc: "Specify a restart command manually. If left unspecified, the service will be stopped and then started"

Reply:
What options to we have for this?

Reply:
well, you can actually define the proper restart command if you want. But such a service, there's no use doing so I guess.